packages/lib-classifier/src/store/TranscriptionReductions/TranscriptionReductions.js

```diff
@@ -0,0 +1,125 @@
+import asyncStates from '@zooniverse/async-states'
+import { flow, getEnv, types } from 'mobx-state-tree'
+
+const CONSENSUS_SCORE_TO_RETIRE = 3
+const MINIMUM_VIEW_TO_RETIRE = 5
+const REDUCER_KEY = 'alice'
+
+const TranscriptionLine = types.model({
+  consensusReached: types.boolean,
+  frame: types.number,
+  hasCollaborated: types.boolean,
+  points: types.array(types.frozen({})),
+  previousAnnotation: types.boolean,
+  textOptions: types.array(types.string),
+})
+
+const TranscriptionReductions = types
+  .model('TranscriptionReductions', {
+    caesarReducerKey: types.frozen(REDUCER_KEY),
+    error: types.maybeNull(types.frozen({})),
+    frame: types.optional(types.number, 0),
```

Comment:
The multi-frame subject viewer does default to the first frame as well, but it's possible that project owners can set a default frame via subject metadata. This is out of scope for the initial feature, but this here should be set to what the `SubjectViewerStore` ends up being set to.

Reply:
0 is the default in the model. It can, of course, be overridden in the snapshot that the Subject model uses to create a new reductions model.